<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 242, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_picture_answer_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>a1_5.jpg</normaloff>a1_5.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_picture_answer_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>a1_6.jpg</normaloff>a1_6.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_picture_answer_1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>a1_1.jpg</normaloff>a1_1.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_picture_answer_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>a1_2.jpg</normaloff>a1_2.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_picture_answer_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>a1_3.jpg</normaloff>a1_3.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_picture_answer_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>a1_4.jpg</normaloff>a1_4.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_skip">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Segoe UI Black&quot;;
color: rgb(128, 172, 207);
background-color: rgb(2, 16, 29);</string>
    </property>
    <property name="text">
     <string>Пропустить</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_count_of_tasks">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
color: rgb(2, 16, 29);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;1/15&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_image">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>331</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>q1.jpg</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>621</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="l_text1">
       <property name="styleSheet">
        <string notr="true">font: 87 10pt &quot;Segoe UI Black&quot;;
color: rgb(2, 16, 29);</string>
       </property>
       <property name="text">
        <string>Выберите правильный ответ и нажмите на него.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_text2">
       <property name="styleSheet">
        <string notr="true">color: rgb(2, 16, 29);
font: 87 10pt &quot;Segoe UI Black&quot;;</string>
       </property>
       <property name="text">
        <string>Если Вы не знаете ответа, то можете нажать  на кнопку &quot;Пропустить&quot;.</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
